### PR TITLE
Fix HTTP request examples

### PR DESCRIPTION
**Pull request by kingthorin**
This PR fixes OWASP/wstg#369.

- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Replace superfluous request (and some response) headers with [...], correct syntax and lead-in text.

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Comment by kingthorin** [comment]
I'm not sure what to do with the cert.org and CMU stuff it was all broken this morning, just read through the link failures. I can roll them back but what a pain in the ash...

File: document/4-Web_Application_Security_Testing/07-Input_Validation_Testing/03-Testing_for_HTTP_Verb_Tampering.md [hunk 1]
```diff
@@ -48,87 +48,87 @@ As the HTML standard does not support request methods other than GET or POST, we
 
 This example is written using the netcat package from openbsd (standard with most Linux distributions). You may also use telnet (included with Windows) in a similar fashion.
 
-1. Crafting custom HTTP requests
+#### 1. Crafting custom HTTP requests
```
[the remark this answers is not in the document]
Comment (kingthorin) [reply document/4-Web_Application_Security_Testing/07-Input_Validation_Testing/03-Testing_for_HTTP_Verb_Tampering.md:51]: It's fine as-is.

[the remark this answers is not in the document]
Comment (kingthorin) [reply document/4-Web_Application_Security_Testing/07-Input_Validation_Testing/03-Testing_for_HTTP_Verb_Tampering.md:51]: Ill tweak the title caps.

**Comment by ThunderSon** [comment]
The CMU stuff should be stable. Could be something that occurred today? I regularly access their KB.

**Comment by kingthorin** [comment]
Agreed, I was really thrown off this morning when our actions started barfing on that stuff. Thanks for the review!

**Pull request merged**